test/tests.bats

```diff
@@ -317,7 +317,7 @@ EOF
 }
 
 @test "checking smtp: rejects spam" {
-  run docker exec mail /bin/sh -c "grep 'Blocked SPAM' /var/log/mail/mail.log | grep external.tld=spam@my-domain.com | wc -l"
+  run docker exec -e SPAMASSASSIN_SPAM_TO_INBOX=0 mail /bin/sh -c "grep 'Blocked SPAM' /var/log/mail/mail.log | grep external.tld=spam@my-domain.com | wc -l"
```

Comment:
This is the other changed line in `tests.bats` that I was referring to in my prior review comment, just marking it so it's more visible in review (_otherwise implicit mentions sometimes get missed accidentally_).